dell/xps/13-9315/default.nix

```diff
@@ -0,0 +1,25 @@
+{lib, ...}: {
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/ssd
+  ];
+
+  # enable firmwares with a license allowing redistribution
+  # this includes the Wi-Fi and Bluetooth firmwares
+  hardware.enableRedistributableFirmware = true;
+
+  # touchpad uses I²C, so PS/2 is unnecessary
+  boot.blacklistedKernelModules = ["psmouse"];
+
+  # enable finger print sensor
+  # configure with `sudo fprintd-enroll <username>`
+  services.fprintd.enable = true;
+  services.fprintd.tod.enable = true;
```

Comment:
I actually was wrong about the default of services.fprintd.tod.driver, it's only an example value.

Reply:
ah! that explains why I couldn't see your comment from the email I received.
Let me rollback